<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportPage</class>
 <widget class="QWidget" name="ReportPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报表与图表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="topBar">
     <layout class="QHBoxLayout" name="topBarLayout">
      <item>
       <widget class="QLabel" name="label_title">
        <property name="text">
         <string>报表中心</string>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="topSpacer"/>
      </item>
      <item>
       <widget class="QComboBox" name="combo_reportType">
        <item>
         <property name="text">
          <string>monthly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>annual</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_year">
        <property name="minimum">
         <number>2000</number>
        </property>
        <property name="maximum">
         <number>2100</number>
        </property>
        <property name="value">
         <number>2025</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_generate">
        <property name="text">
         <string>生成报表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_export">
        <property name="text">
         <string>导出 PDF/CSV</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>

     <widget class="QWidget" name="leftPanel">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QListView" name="list_reportSections"/>
       </item>
     </layout>
     </widget>

     <widget class="QWidget" name="rightPanel">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QWidget" name="chartContainer">
         <!-- 这里填充 QChartView -->
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="text_reportSummary">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
